src/btree/bt_split.c

```diff
@@ -1788,6 +1790,10 @@ __wt_multi_to_ref(WT_SESSION_IMPL *session,
 
 		WT_RET(__wt_calloc_one(session, &ref->page_las));
 		*ref->page_las = multi->page_las;
+		ref->page_las->birthmarks = multi->page_las.birthmarks;
+#ifdef HAVE_DIAGNOSTIC
+		ref->page_las->birthmarks_cnt = multi->page_las.birthmarks_cnt;
+#endif
```

Comment:
@sulabhM, I don't understand this code, can you please explain why it's a diagnostic?

Reply:
The birthmarks_cnt is diagnostic because it is not really needed at the moment for a functional reason. It is just to test the code for correction. At the moment, birthmarks_cnt is always going to be the same as the number of keys on the page, which we determine by going over the LAS file. Certainly, this will change when we stop initialising from LAS, also when we add optimisations around not keeping birthmark per key.